includes/rules/missing_headings.php

```diff
@@ -13,12 +13,17 @@
  * @return array
  */
 function edac_rule_missing_headings( $content, $post ) {
+	if ( ! EDAC_KEY_VALID ) {
+		return;
+	}
 
-	$dom = str_get_html( $post->post_content );
-	if ( empty( $dom ) ) {
-		goto error;
+	$word_count = str_word_count( strip_tags( $post->post_content ) );
+
+	if ( $word_count <= 400 ) {
```

Comment:
I think this should only have a less-than sign. The issue called for fewer than 400 words.
"I don't think this should flag if there are fewer than 400 words on the page."